# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: layout/generic/crashtests/crashtests.list
# Commit: 83b018946c0b
# Full Hash: 83b018946c0b78a2035e6d50c3ccd55eac3570a4
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2022-06-01 04:09:30
# Description:
#   Bug 1771556 - Handle backdrop capture failing clip check when sub-graph is visible r=gfx-reviewers,lsalzman
#   
#   This can happen when there is a long nested chain of backdrop-filters
#   and float inaccuracies cause the final capture primitive to be culled.
#   
# ==============================================================================

diff -r d2a56d51c65b -r 83b018946c0b layout/generic/crashtests/crashtests.list
--- a/layout/generic/crashtests/crashtests.list	Tue May 31 22:21:15 2022 +0000
+++ b/layout/generic/crashtests/crashtests.list	Tue May 31 22:55:32 2022 +0000
@@ -338,7 +338,7 @@
 load 477928.html
 load 478131-1.html
 load 478170-1.html
-asserts(1591-3581) load 478185-1.html
+asserts(1591-3980) load 478185-1.html
 load 478504.html
 asserts-if(!Android,0-1) load 479938-1.html # Bug 575011
 load 480345-1.html
@@ -508,7 +508,7 @@
 load 812822-1.html
 load 812879-1.html
 load 812879-2.html
-asserts-if(winWidget,4-42) load 812893.html # bug 1581653
+asserts-if(winWidget,0-42) load 812893.html # bug 1581653
 load 814995.html
 load 822910.xhtml
 load 824297-1.html
@@ -531,7 +531,7 @@
 load 847211-1.html
 load 849603.html
 load 849987.html
-asserts(34-46) load 850931.html # nested multicols, inner multicol has column-width 1 app unit ...
+asserts(8-46) load 850931.html # nested multicols, inner multicol has column-width 1 app unit ...
 load 851396-1.html
 load 854263-1.html
 load 862185.html
@@ -728,7 +728,7 @@
 load 1544060-1.html
 load 1544060-2.html
 pref(layout.css.grid-template-subgrid-value.enabled,true) load 1553824.html
-asserts(26-34) load 1554824.html # extreme sizes, column-width: 0em
+asserts(26-42) load 1554824.html # extreme sizes, column-width: 0em
 load 1555142.html
 load 1560349.html
 load 1560397.html
@@ -777,7 +777,7 @@
 load 1640028.html
 load 1640051.html
 load 1640275.html
-pref(layout.accessiblecaret.enabled,true) asserts(16-23) load 1644819.html # bug 1688115
+pref(layout.accessiblecaret.enabled,true) asserts(16-26) load 1644819.html # bug 1688115
 load 1645549-1.html
 load 1648577.html
 load 1652618.html
